cypress/support/fragments/invoices/invoices.js

```diff
@@ -6,10 +6,14 @@ const buttonNew = Button('New');
 const saveAndClose = Button('Save & close');
 const invoiceCreatedMessage = 'Invoice has been saved';
 const invoiceLineCreatedMessage = 'Invoice line has been saved';
+const InvoiceApprovedMessage = 'Invoice has been approved successfully';
+const InvoicePaidMessage = 'Invoice has been paid successfully';
```

Comment:
May be better to create constant object:

```
invoiceStates = {
  approved: 'Invoice has been approved successfully',
  paind: 'Invoice has been paid successfully'
...
}
```

Reply:
Updated